uint8 agentId
float64 confidence
float64[3] agentPosition
float64[3] agentVelocity
